Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Scripts\flask.exe\__main__.py", line 7, in <module>
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\flask\cli.py", line 1129, in main
    cli.main()
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\click\core.py", line 1082, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\click\core.py", line 1697, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\click\core.py", line 1443, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\click\core.py", line 788, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\click\decorators.py", line 92, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\click\core.py", line 788, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\flask\cli.py", line 977, in run_command
    raise e from None
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\flask\cli.py", line 961, in run_command
    app: WSGIApplication = info.load_app()  # pyright: ignore
                           ^^^^^^^^^^^^^^^
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\flask\cli.py", line 353, in load_app
    app = locate_app(import_name, None, raise_if_not_found=False)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\venv\Lib\site-packages\flask\cli.py", line 245, in locate_app
    __import__(module_name)
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\app.py", line 12, in <module>
    from controller import * 
    ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\controller\chibo_controller.py", line 2, in <module>
    from model.chibo_model import chibo_model
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\model\chibo_model.py", line 3, in <module>
    from database import Database
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\database.py", line 53, in <module>
    db = Database()
         ^^^^^^^^^^
  File "C:\Users\84765\Desktop\ThucTapChuyenMon\flask_app\database.py", line 20, in __init__
    print("\u2705 K\u1ebft n\u1ed1i c\u01a1 s\u1edf d\u1eef li\u1ec7u thành công!")
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2544.0_x64__qbz5n2kfra8p0\Lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'charmap' codec can't encode character '\u2705' in position 0: character maps to <undefined>
